src/test/run-pass/allocator-override.rs

```diff
@@ -10,6 +10,7 @@
 
 // no-prefer-dynamic
 // aux-build:allocator-dummy.rs
+// ignore-emscripten
```

Comment:
Should these allocator tests still be ignored for emscripten? If it's jemalloc-related issues I'd expect to also see `// ignore-windows` which doesn't have jemalloc today.